elliptic-curve/src/public_key.rs

```diff
@@ -436,6 +458,7 @@ where
         let public_key_bytes = self.to_encoded_point(false);
         let subject_public_key = der::asn1::BitStringRef::new(0, public_key_bytes.as_bytes())?;
 
+        // TODO: use `encode_as_der` here?
```

Comment:
The `der::Document` created here only has `TryFrom<Vec<u8>>` and not `TryFrom<[u8]>`, so using `encode_as_der` did not seem feasible. Bit of a hassle.